.gitignore

```diff
@@ -84,6 +84,7 @@ _ts4.2
 # the downlevel-dts output folders
 _ts3.4
```

Comment:
Can we get rid of `_ts3.4` now?

Reply:
we should keep it in gitignore until we merge it, as it will prevent us from accidently pushing those files while switching between v6 and main